File: HomeworkDay06/app/src/main/java/com/example/homework_day_06/HomeworkActivity.java
package com.example.homework_day_06;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Color;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;

import com.example.homework_day_06.fragment.HomeFragment;
import com.example.homework_day_06.fragment.MainFragment;
import com.example.homework_day_06.fragment.MineFragment;

public class HomeworkActivity extends AppCompatActivity {

    private static final String TAG = "HomeworkActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_homework);

        getSupportFragmentManager().beginTransaction()
                .setReorderingAllowed(true)
                .add(R.id.page_fragment_container_view, HomeFragment.class, null, "BUTTON_FRAGMENT_TAG")
                .commit();

        Button buttonToPageHome = findViewById(R.id.to_page_home);
        Button buttonToPageMine = findViewById(R.id.to_page_mine);

        buttonToPageHome.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Log.d(TAG, "to_page_home");
                getSupportFragmentManager()
                        .beginTransaction()
                        .setReorderingAllowed(true)
                        .replace(R.id.page_fragment_container_view, HomeFragment.class, null, "BUTTON_FRAGMENT_TAG")
                        .commit();
                buttonToPageHome.setBackgroundColor(getResources().getColor(R.color.coral));
                buttonToPageMine.setBackgroundColor(getResources().getColor(R.color.gray));
            }
        });

        buttonToPageMine.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Log.d(TAG, "to_page_mine");
                getSupportFragmentManager()
                        .beginTransaction()
                        .setReorderingAllowed(true)
                        .replace(R.id.page_fragment_container_view, MineFragment.class, null, "BUTTON_FRAGMENT_TAG")
                        .commit();
                buttonToPageHome.setBackgroundColor(getResources().getColor(R.color.gray));
                buttonToPageMine.setBackgroundColor(getResources().getColor(R.color.coral));
            }
        });
    }
}

File: HomeworkDay11/app/src/main/java/com/example/homework_day_11/HomeworkActivity.kt
package com.example.homework_day_11

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.example.homework_day_11.fragment.HomeFragment
import com.example.homework_day_11.fragment.MineFragment

class HomeworkActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_homework)

        supportFragmentManager.beginTransaction()
            .setReorderingAllowed(true)
            .add(
                R.id.pageFragmentContainerView,
                HomeFragment::class.java,
                null,
                "BUTTON_FRAGMENT_TAG"
            )
            .commit()

        val buttonToHome = findViewById<Button>(R.id.toPageFragmentHome)
        val buttonToMine = findViewById<Button>(R.id.toPageFragmentMine)
        buttonToHome.setOnClickListener {
            supportFragmentManager.beginTransaction()
                .setReorderingAllowed(true)
                .replace(
                    R.id.pageFragmentContainerView,
                    HomeFragment::class.java,
                    null,
                    "BUTTON_FRAGMENT_TAG"
                )
                .commit()
            buttonToHome.setBackgroundColor(resources.getColor(R.color.coral))
            buttonToMine.setBackgroundColor(resources.getColor(R.color.gray))
        }
        buttonToMine.setOnClickListener {
            supportFragmentManager.beginTransaction()
                .setReorderingAllowed(true)
                .replace(
                    R.id.pageFragmentContainerView,
                    MineFragment::class.java,
                    null,
                    "BUTTON_FRAGMENT_TAG"
                )
                .commit()
            buttonToHome.setBackgroundColor(resources.getColor(R.color.gray))
            buttonToMine.setBackgroundColor(resources.getColor(R.color.coral))
        }
    }
}